developing/README.md

```diff
@@ -0,0 +1,196 @@
+## Setting up a full dev environment
+
+If you want to work on Proxstar using a 1:1 development setup, there are a couple things you're going to need
+
+- A machine you can SSH into, portforward from, and run Flask, Redis, and Docker on
+- At least one (1) Proxmox host running Proxmox >6.3
+- Docker
+- SSH portforwarding
+- A CSH account
+- An RTP (to tell you secrets)
+
+1. Configure your Proxmox node
+
+I would recommend setting up a development account on your Proxmox node. Name it anything. (Maybe `proxstartest`?). This is necessary to grab authentication tokens and the like. It should have the same permissions as `root@pam`. You can accomplish this by creating a group in `Datacenter > Permissions > Groups` and adding `Administrator` permissions to the group, then adding your user to the group. If you do this, then it's easy to enable/disable it for development. You should also generate an SSH key for the user.
+
+You will also have to set up a pool on your Proxmox node with your csh username. To do this, go into `Datacenter > Permissions > Pools > Create`.
+
+2. Set up your environment
+
+If you're trying to run this all on a VM without a graphical web browser, you can forward traffic to your computer using SSH.
+```
+ssh example@dev-server.csh.rit.edu -L 8000:localhost:8000
+```
+
+Clone down the repository, and create a Virtualenv to do your work in.
+```
+mkdir venv
+python3.8 -m venv venv
```

Comment:
Do we require 3.8? Maybe that should be in requirements

Reply:
I'm just thinking it could be noted here: https://github.com/ComputerScienceHouse/proxstar/pull/114/files/8507ac75c22cda09eafeee2885bfa7867442b9c6#diff-178576a49b60e49d3fd9657734b71632e85e420dc8f8b84c5032f6c827687431R5-R10

Reply:
Honestly though I could see it going either way, since none of those are really software requirements either...

Reply:
If it runs on 3.8 bump it all to 3.8. 3.6 is EOL in December.